### PR TITLE
Expansion panel filters

### LINKED ISSUE #295: Make profile+needs selectors collapsable on mobile

### PR DESCRIPTION
**Pull request by smcmurtry**
closes #295
to test try resizing the screen above and below 600px

### CODE REVIEW

File: components/needs_selector.js [hunk 1]
```diff
@@ -23,12 +37,23 @@ const styles = theme => ({
...
   clearButton: {
-    textAlign: "right",
     textDecoration: "underline"
+  },
+  gridItemButton: {
+    textAlign: "center"
   }
 });
 
 class NeedsSelector extends Component {
```
Comment (sastels) [inline components/needs_selector.js:46]: We should probably have a test that the new NeedsSelector and ProfileSelector behave appropriately under big / small widths.

File: components/needs_selector.js [hunk 2]
```diff
@@ -23,12 +37,23 @@ const styles = theme => ({
...
+  state = {
+    open: true
```
Comment (sastels) [inline components/needs_selector.js]: Should the default state be closed on mobile? Probably worth checking with Raluca or Michelle.

Comment (smcmurtry) [reply components/needs_selector.js]: changed default mobile state to closed based on discussion w/ Raluca

Comment (sastels) [reply components/needs_selector.js]: (might be good to push those changes 😛 )

**Review by sastels — APPROVED** [review]
🦑

**Review by maxneuvians — APPROVED** [review]
Well done! 🦑

**Pull request merged**